core/src/main/scala/org/apache/spark/SparkEnv.scala

```diff
@@ -77,6 +76,12 @@ class SparkEnv (
 
   def shuffleManager: ShuffleManager = _shuffleManager
 
+  // We initialize the MemoryManager later in SparkContext after DriverPlugin is loaded
+  // to allow the plugin to overwrite memory configurations
+  private var _memoryManager: MemoryManager = _
```

Comment:
nit: move the definition along with `_shuffleManager` above ?

Reply:
sure will do.